app/controllers/publications_controller.rb

```diff
@@ -101,8 +101,8 @@ def update
       head :gone
       return
     end
-    if old_pub.sciencewire_id.present? || old_pub.pmid.present? || old_pub.wos_uid.present?
-      render json: { "error": 'This record may not be modified.  If you had originally entered details for the record, it has been superceded by a central record.' }, status: :forbidden, format: 'json'
+    if !old_pub.cap_pub? # only manually entered publications may be updated like this
```

Comment:
NOT?

Reply:
PR LGTM but it'd be good to have this question answered before we merge. There seems to be an important notion here, that of whether a publication is "manual" or "manually entered" but this is a case where the language of the code does not match the language of the domain, so a reviewer without intimate sul_pub knowledge can not tell if this line is doing EXACTLY what you want or EXACTLY what you want to avoid. Of course, I trust you as the code author, but I also know that we all make silly typos and mistakes like getting boolean conditions the exact opposite, so ... not merging for now, but leaving this long-winded comment which someone should put on a T-shirt sometime.